viite-UI/src/view/ProjectLinkLayer.js

```diff
@@ -128,7 +128,8 @@
       layer: [vectorLayer, suravageRoadProjectLayer],
       condition: ol.events.condition.singleClick,
       style: function(feature, resolution) {
-        if (feature.projectLinkData.status === notHandledStatus || feature.projectLinkData.status === newRoadAddressStatus || feature.projectLinkData.roadLinkSource === 3){
+        if (feature.projectLinkData.status === notHandledStatus || feature.projectLinkData.status === newRoadAddressStatus ||
```

Comment:
We should probably replace this with a utility method for checking the status. These status checks seem to get common and could be replaced with a function projectLinkStatusIn(projectLink, [allowedStatus1, allowedStatus2, ...]).